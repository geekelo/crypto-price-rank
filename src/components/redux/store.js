import { configureStore } from '@reduxjs/toolkit';
import themeSlice from './theme/themeSlice';
import cryptosSlice from './crypto/cryptosSlice';

const store = configureStore({
  reducer: {
    theme: themeSlice,
    cryptos: cryptosSlice,
  },
});

export default store;
