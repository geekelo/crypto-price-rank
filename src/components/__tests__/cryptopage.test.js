import React from 'react';
import renderer from 'react-test-renderer'; // for creating snapshots
import { Provider } from 'react-redux';
import { MemoryRouter, Route, Routes } from 'react-router-dom';
import store from '../redux/store';
import Cryptopage from '../cryptopage';

test('Cryptopage component matches snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MemoryRouter initialEntries={['/1']}>
        <Routes>
          <Route path="/:id" component={Cryptopage} />
         </Routes>
      </MemoryRouter>
    </Provider>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
