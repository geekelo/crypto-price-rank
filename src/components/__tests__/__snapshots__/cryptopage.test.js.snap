// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cryptopage component matches snapshot 1`] = `null`;
